console/src/app/pages/actions/add-action-dialog/add-action-dialog.component.ts

```diff
@@ -26,12 +29,15 @@ export class AddActionDialogComponent {
   public id: string = '';
 
   public opened$ = this.dialogRef.afterOpened().pipe(mapTo(true));
-
+  public form!: UntypedFormGroup;
```

Comment:
```suggestion
  public form: FormGroup = new FormGroup({
    name: new FormControl<string>('', []),
    script: new FormControl<string>('', []),
    durationInSec: new FormControl<number>(10, []),
    allowedToFail: new FormControl<boolean>(false, []),
  });
```
can we initialize this with typedforms and remove the fixed variables on top?

Reply:
Absolutely!